Makefile

```diff
@@ -87,6 +87,15 @@ push-operator: build-operator
 	$(CONTAINER_CMD) tag operator:forklift-operator-image ${OPERATOR_IMAGE}
 	$(CONTAINER_CMD) push ${OPERATOR_IMAGE}
 
+# Build the docker image
+build-imageio:
+	bazel run ovirt-imageio/ovirt-imageio-image
+
+# Push the docker image
+push-imageio: build-operator
```

Comment:
the term `imageio` is not that good in this context - how about `ovirt-populator` instead?